app/src/main/java/org/schabi/newpipe/info_list/holder/CommentsMiniInfoItemHolder.java

```diff
@@ -120,6 +126,21 @@ public void updateFromItem(final InfoItem infoItem, final HistoryRecordManager h
                 itemBuilder.getOnCommentsSelectedListener().selected(item);
             }
         });
+
+
+        itemView.setOnLongClickListener(new View.OnLongClickListener() {
+            @Override
+            public boolean onLongClick(View view) {
+
+                ClipboardManager clipboardManager = (ClipboardManager) itemBuilder.getContext()
+                                                    .getSystemService(Context.CLIPBOARD_SERVICE);
+                clipboardManager.setPrimaryClip(ClipData.newPlainText(null,itemContentView.getText()));
```

Comment:
`itemContentView.getText()` can sometimes copy the ellipsized text. Use `commentText` instead.
```suggestion
clipboardManager.setPrimaryClip(ClipData.newPlainText(null,commentText));
```